meteor/client/ui/Settings/ConfigManifestSettings.tsx

```diff
@@ -80,6 +81,65 @@ function filterLayerMappings(
 	return result
 }
 
+function getFilteredGfxDefaultTableValues<DBInterface extends { _id: ProtectedString<any> }>(
+	item: ConfigManifestEntryFilterDefaultsFromShowMapping<boolean>,
+	configPath: string,
+	object: DBInterface,
+	alternateObject?: any
```

Comment:
`object` seems like a non-descriptive name for a database object type.

Reply:
I'm considering changing `object` to `showStyleBaseDBObject` and `alternateObject` to `showStyleVariantDBObject`. If you have a better suggestion please let me know!